### DISCUSSION TITLE
Choosing between Record and Table

### DISCUSSION THREAD

**Discussion by JLPLabs** [pallene-lang/pallene#476] [Q&A] [answered]
How to you decide between using a Record and a Table? 
Many thanks.

**Comment by hugomg** [comment, marked as answer]
Records are implemented as userdata. Creating and acessing a record is faster in Pallene.

The downside of records is that they are opaque to Lua. Lua cannot create records and also cannot directly access the fields.

**Reply by JLPLabs** [reply]
Thanks again for the quick reply.
